src/modules/reminder/dto/reminder-request-body.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import { IsDate, IsEmail, IsNotEmpty, IsString } from 'class-validator';
+
+export class ReminderRequestBodyDto {
```

Comment:
Remider Request Body для чего? Для создания? Обновления? Удаления?
```suggestion
export class ReminderCreateRequestBodyDto {
```